pkg/kubernetes/podtemplatespec/v1alpha1/podtemplatespec.go

```diff
@@ -0,0 +1,278 @@
+// Copyright © 2018-2019 The OpenEBS Authors
```

Comment:
Let us remove spec from folder and file name. What do you suggest.

Reply:
There are two different struct named `corev1.PodTemplateSpec` and `corev1.PodTemplate` in kubernetes. It will create confusion as deployment accepts `corev1.PodTemplateSpec` and not `corev1.PodTemplate`.